openpype/scripts/otio_burnin.py

```diff
@@ -144,7 +185,13 @@ def __init__(
             self.options_init.update(options_init)
 
     def add_text(
-        self, text, align, frame_start=None, frame_end=None, options=None
+        self,
+        text,
+        align,
+        frame_start=None,
+        frame_end=None,
+        options=None,
+        cmd=""
```

Comment:
Consider this....
```suggestion
        cmd=None
```

Reply:
> Could I ask why?
`cmd = None` works the same as with `cmd = ""`. 
It's more like preference on style